pkg/epp/datalayer/metrics/logger.go

```diff
@@ -95,8 +96,8 @@ func printDebugMetrics(logger logr.Logger, datastore datalayer.PoolInfo, stalene
 	freshPods := datastore.PodList(podsWithFreshMetrics(stalenessThreshold))
 	stalePods := datastore.PodList(podsWithStaleMetrics(stalenessThreshold))
 
-	logger.V(logutil.VERBOSE).Info("Current Pods and metrics gathered",
-		"Fresh metrics", freshPods, "Stale metrics", stalePods)
+	s := fmt.Sprintf("Current Pods and metrics gathered. Fresh metrics: %+v, Stale metrics: %+v", freshPods, stalePods)
+	logger.V(logutil.VERBOSE).Info(s)
```

Comment:
VERBOSE is too high for this logging message. we should go with TRACE.
(wasn't changed by this PR, but caught in the review)